src/scripts/farming/aura/Aura.ts

```diff
@@ -0,0 +1,68 @@
+class Aura {
+
+    auraType: AuraType;
+    auraMultipliers: number[];
+
+    constructor(auraType: AuraType, auraMultipliers: number[]) {
+        this.auraType = auraType;
+        this.auraMultipliers = auraMultipliers;
+    }
+
+    /**
+     * Handles applying the berry's Aura to its neighbors
+     */
+    applyAura(index: number): void {
+        const plot = App.game.farming.plotList[index];
+        console.log(index, plot, plot.stage());
+        if (plot.stage() < PlotStage.Taller) {
+            return;
+        }
+        const multiplier = this.auraMultipliers[plot.stage() - 2];
+        switch (this.auraType) {
+            case AuraType.Attract:
+            case AuraType.Egg:
+                const currentMultiplier = App.game.farming.externalAuras[this.auraType]();
+                App.game.farming.externalAuras[this.auraType](currentMultiplier * multiplier);
+                break;
+            default:
+                const plots = Plot.findNearPlots(index);
+                console.log(plots);
+                for (const nearIdx of plots) {
+                    const nearPlot = App.game.farming.plotList[nearIdx];
+                    nearPlot.addAura(this.auraType, multiplier);
+                }
+                break;
+        }
+    }
+
+    getLabel(stage: PlotStage): string {
+        if (stage < PlotStage.Taller) {
+            return '';
+        }
+        let label = '';
+        switch (this.auraType) {
+            case AuraType.Attract:
+                label = 'Attract: ';
```

Comment:
This entire switch statement looks like it can be replaced with
```js
let label = `${AuraType[this.auraType]}: `;
```
Maybe a `GameHelper.humanifyString` if there might be a multiword AuraType added in the future.